src/utils/sha_256/packed_sha256.cairo

```diff
@@ -0,0 +1,287 @@
+from starkware.cairo.common.alloc import alloc
```

Comment:
This is from Cartridge/Starkware implementation

Reply:
That's true, completely agree. I'll try to research on how we can implement this 👍